CMakeLists.txt

```diff
@@ -98,31 +100,6 @@ if (NOT TARGET xeus AND NOT TARGET xeus-static)
     endif()
 endif ()
 
-set(CppInterOp_REQUIRED_VERSION 1.3.0)
-
-# Flags
-# =====
-include(CheckCXXCompilerFlag)
-
-if (MSVC)
-  add_compile_options(/wd4251 /wd4141)
-  add_compile_options(/wd4018 /wd4267 /wd4715 /wd4146 /wd4129)
-  add_compile_options(/EHsc)
-else()
-  add_compile_options(-fexceptions)
-endif ()
-
-if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Intel")
-    if(NOT EMSCRIPTEN)
-        add_compile_options(-Wunused-parameter -Wextra -Wreorder)
-    endif()
-endif ()
-
-if(EMSCRIPTEN)
-    # ENV (https://github.com/emscripten-core/emscripten/commit/6d9681ad04f60b41ef6345ab06c29bbc9eeb84e0)
-    set(EMSCRIPTEN_FEATURES "${EMSCRIPTEN_FEATURES} -s \"EXTRA_EXPORTED_RUNTIME_METHODS=[ENV']\"")
-endif()
-
```

Comment:
These are redundant as are being addressed through `macro(xeus_cpp_set_common_options target_name)` so can be totally removed from here.

Reply:
I think some flags are still missing in the macro (-fexceptions and /EHsc, I may have missed others).

Reply:
Hmm, having a look !

Reply:
Yupp those were the only ones. Should be ready now !